src/SerEvaluator.cs

```diff
@@ -711,6 +679,10 @@ private SerConfig GetNewJson(UserParameter parameter, string userJson, string wo
             if (ondemandObject != null)
                 parameter.OnDemand = serConfig["onDemand"]?.ToObject<bool>() ?? false;
 
+            var configConnection = JObject.Parse(JsonConvert.SerializeObject(mainConnection, Formatting.Indented));
+            configConnection["credentials"]["cert"] = null;
```

Comment:
why you don't use the dynamic way. configConnection.credential.cert ??

Reply:
Is Implemented